src/NzbDrone.Core/Organizer/FileNameSampleService.cs

```diff
@@ -88,15 +88,19 @@ public FileNameSampleService(IBuildFileNames buildFileNames)
             var mediaInfo = new MediaInfoModel()
             {
                 VideoCodec = "AVC",
+                VideoBitDepth = "8",
                 AudioFormat = "DTS",
+                AudioChannels = "5.1",
```

Comment:
Audio channels is a whole number, which would be `6` in this case and `AudioChannelPositions` would be set to `3/2/0.1`, which would get calculated to 5.1.